geom/src/polyline.rs

```diff
@@ -123,6 +123,12 @@ impl PolyLine {
         PolyLine::must_new(pts)
     }
 
+    pub fn quadrant(&self) -> i64 {
+        let mut line_angle: f64 = self.overall_angle().normalized_radians();
```

Comment:
The `mut` isn't needed here. Below you're re-declaring the variable by doing `let` again -- see https://doc.rust-lang.org/rust-by-example/variable_bindings/scope.html. So you could either remove the `mut` or the `let` below -- both equivalent, I don't think there's much of a style difference.